### ISSUE TITLE
POSIX compatible tool owcc fails on Windows 10 x64

### ISSUE THREAD

**Issue by Mazdaywik** [open-watcom/open-watcom-v2#219]
I want compile project with two files:

```C++
// file foo.cpp
#include <stdio.h>

int foo(int x);

int main() {
  printf("foo(%d) = %d\n", 10, foo(10));
  return 0;
}
```

```C++
// file bar/baz.cpp
int foo(int x) {
  return x * x;
}
```

I use Windows 10 x64. I compile with this command line:

```
owcc foo.cpp bar/baz.cpp
```

When I use x64 tool, I got success compilation (valid exe created) but have error message:

![watcom](https://cloud.githubusercontent.com/assets/1309859/11920278/6652098e-a77b-11e5-9365-8356d989c2a8.png)
(Program "Open Watcom C/C++ Compile and Link Utility" is stopped, Windows searches way to fix the error)

and file `__owcc__.lnk` is leaving in current directory.

When I use x86 tool, I got success silent compilation and file `__owcc.lnk` not exist in current directory.

**Comment by jmalak** [comment]
Thanks for youre bug report.
It looks like final cleanup is broken because linker command file is not deleted for some reason.
I will check. May be related to use of directory with forward slash for source.

**Comment by jmalak** [comment]
it is fixed in git repository by 3cf94ebc6dc84df1eb70994c82605b8c60e8fe9b .
You need to recompile it from latest source code.

**Comment by jmalak** [comment]
The problem was more complex that more fixes were necessary.
The problem was not specific to Windows 10, but was masked on other platforms due to different memory management.
Now should be working without crash on any supported host(16/32/64-bit).
